e2e-tests/desktop/fixtures/electronTest.js

```diff
@@ -6,14 +6,21 @@
 import { test } from '@playwright/test';
 import path from 'path';
 import fs from 'fs';
+import { fileURLToPath } from 'url';
 
 /**
  * Get the path to the Boundary Desktop executable
  * @param buildDirectory
  * @return {string}
  */
 const getExecutablePath = (buildDirectory = 'out') => {
-  const rootDir = path.resolve('../ui/desktop/electron-app');
+  // Using process.cwd() can change depending on where you run the tests so we use the current file location
+  // __dirname isn't available in ES modules so we need to indirectly get the directory
+  // TODO: Switch to `import.meta.dirname` when we switch to Node.js 20+
```

Comment:
Since the decouple of e2e test with the UI's, which was a good call, I do think e2e can diverge on what node version they run. In other words, I do think e2e test should use node 22 or even 24.
The e2e tests are not part of any related UI automation or procedure. I think using latest version can benefit more than harm the e2e suite.
The only "con" I see, is the developer running them might need to change the node version within its environment, since we work most of our time on node20, but that's it.

What are your thoughts? 

Reply:
Yeah I could make it a requirement to use a more recent node. I wasn't sure what others were still running so I didn't want to have people switch node versions if people were still using <20 but I have no problems with making it a requirement and using newer stuff.

24 doesn't seem to be in LTS yet (or even current) so I think 22 would be the highest we should go. The property was available starting in node 20 so I could just require >= 20 since we're about to upgrade our node versions across our CI and already support it in our apps if that's the case, I don't mind changing this

Reply:
Yep, aiming for LTS is the right way. I mentioned v24 as an example of if running latest benefits the e2e suite, due to a new feature or something similar I am down for it 😉 .

The switching versions, in this case I think its totally fine, we are doing it for the benefit of the e2e, also as you mention, as soon we are unblock with the windows runner we will roll out node upgrade anyways.

Reply:
Updated to use 20 and removed the extra code for versions below node 20